Tests/TuistGeneratorTests/Generator/BuildPhaseGeneratorTests.swift

```diff
@@ -580,6 +580,7 @@ final class BuildPhaseGeneratorTests: TuistUnitTestCase {
         XCTAssertEqual(postBuildPhase.shellPath, "/bin/sh")
         XCTAssertEqual(postBuildPhase.shellScript, "\"${PROJECT_DIR}\"/script.sh arg")
         XCTAssertFalse(postBuildPhase.showEnvVarsInLog)
+        XCTAssertTrue(postBuildPhase.alwaysOutOfDate)
```

Comment:
👍 

Minor: I would perhaps also explicitly verify that `preBuildPhase.alwaysOutOfDate` is `nil` in the section above - to ensure we are indeed keeping that flag unset when not specified.

Reply:
Unfortunately the `XcodeProj` version does indeed set a default already, rather than using `nil`, so I cannot exactly do that check, but I've added a line that makes sure at least that default (`false`) is set in case the user doesn't specify anything. See https://github.com/tuist/XcodeProj/blob/master/Sources/XcodeProj/Objects/BuildPhase/PBXShellScriptBuildPhase.swift#L61